src/content/decentralized-identity.md

```diff
@@ -0,0 +1,157 @@
+
+## Benefits of decentralized identity {#benefits-of-decentralized-identity} 
+
+1. Decentralized identity increases increases individual control of identifying information. Decentralized identifiers and attestations can be verified without relying on centralized authorities and third-party services. 
+
+2. Decentralized identity solutions facilitates a trustless, seamless, and privacy-protecting method for verifying and managing user identity. 
+
+3. Decentralized identity harnesses blockchain technology, which creates trust between different parties and provides cryptographic guarantees to prove the validity of attestations.
+
+4. Decentralized identity makes identity data portable. Users store attestations and identifiers in mobile wallet and can share with any party of their choice. Decentralized identifiers and attestations are not locked into the database of the issuing organization. 
+
+## Decentralized identity use-cases {#decentralized-identity-use-cases}
+
+Decentralized identity has many potential use-cases:
+
+### 1. Universal dApp logins {#universal-dapp-logins} 
+[Decentralized applications](/dapps/) can issue attestations to users, which can be stored in an Ethereum wallet. An attestation in this case could be an an [NFT](/nft/) granting the holder access to an online community. 
+
+A [Sign-In With Ethereum](https://eips.ethereum.org/EIPS/eip-4361) function would then enable servers to confirm the user's Ethereum account and fetch the required attestation from their account address. This means users can access the dapp without inputting passwords or sharing personal information.  
+
+### 2. KYC Authentication {#kyc-authentication}
+Using many online services requires individuals to provide attestations and credentials, such as a driving license or national passport. But this approach is problematic because private user information can be compromised and service providers cannot verify the authenticity of the attestation. 
+
+Decentralized identity allows companies to skip on conventional Know-Your-Customer (KYC) processes and authenticate user identities via Verifiable Credentials. This reduces the cost of identity management and prevents the use of fake documentation. 
+
+### 3. Voting and online communities {#voting-and-online-communities}
+
+Online voting and social media are two novel applications for decentralized identity. Online voting schemes are susceptible to manipulation, especially if malicious actors create false identities to vote. Asking individuals to present on-chain attestations can improve the integrity of online voting processes. 
+
+Decentralized identity can enable the creation of online communities that are free of fake accounts. Each user must authenticate their identity using an on-chain identity system, like the Ethereum Name Service, reducing the possibility of bots overrunning social platforms. 
+
+## Use decentralized identity applications on Ethereum
```

Comment:
I think we can drop 'on Ethereum here'. Can we also drop 'applications'? I think so... maybe it's technically less correct.

```suggestion
## Use decentralized identity {#use-decentralized-identity}
```

```suggestion
## Use decentralized identity applications {#use-decentralized-identity}
```

Reply:
Yup, saying "use decentralized identity" would likely be technically incorrect. Think of those projects mentioned as "identity-as-a-service" solutions. Here's a definition of IDaaS for context: 

*"Identity-as-a-service, or IDaaS, is an application delivery model (like software-as-a-service, or SaaS) that allows users to connect to and use identity management services from the cloud."*